File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {


        while(nums.size() != 1){
            for(int i = 0 ; i < nums.size()-1 ; i++){
                nums[i] = (nums[i]+nums[i+1])%10;
            }
            nums.pop_back();
        }

        return nums[0];


    }
};

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.java
class Solution {
    public int triangularSum(int[] nums) {

        int l = nums.length - 1;

        while (l >= 1) {
            for (int i = 0; i < l; i++) {
                nums[i] = ((nums[i] + nums[i + 1]) % 10);
            }
            l--;
            // System.out.println(Arrays.toString(nums));
        }

        return nums[0];

    }
}